lib/api/api.js

```diff
@@ -87,9 +89,11 @@ const api = {
     bucketDelete,
     bucketGet,
     bucketGetACL,
+    bucketGetVersioning,
```

Comment:
I don't think we need the integration test wrt policies because only the bucket owner can perform these actions.